app/controllers/rating_votes_controller.rb

```diff
@@ -1,5 +1,8 @@
 class RatingVotesController < ApplicationController
   after_action :verify_authorized
+  after_action only: [:create] do
+    Audit::Logger.log(:moderator, current_user, params.dup)
```

Comment:
shold we keep the `current_user.auditable? check here?

Reply:
AFAIK this action is only possible if you are a moderator.